### ISSUE TITLE
I comming !

### ISSUE THREAD

**Issue by rqhua** [1187981465/Note#1]
好开心 我在你家的一楼

**Comment by FYH-fly** [comment]
厉害了， 最原始的聊天方式了。 我想去写个插件去

**Comment by rqhua** [comment]
去啊 我等你飞升

**Comment by rqhua** [comment]
公司开会时偶然坐女神旁边，总在想用什么话开口。突然外面下起了大雨。很大很大，我问她下雨了你一会儿怎么回去。她说我男朋友一会儿开奔驰来接我……我……心里酸溜溜地想：开车接就开车呗，何必非要把奔驰二字说出来？！

**Comment by FYH-fly** [comment]
我感觉也很崩溃， 没办法上微信没有办法上QQ，更没办法吹牛逼，能不能把田总也给弄过来。

**Comment by rqhua** [comment]
Every day and every,there is a SB called JunZhang always works on the internet,and he says "看那谁好没文化啊" and "还用中文呢",what can i do  save him!

**Comment by FYH-fly** [comment]
president zhang is a fool and so funny, I can't speak with him, so I quit that group.

**Comment by Sirlu** [comment]
hello world！The best language is PHP!

**Comment by Sirlu** [comment]
I Looking your kidding in the dark

**Comment by rqhua** [comment]
yeah , just because he is SB,and live in a darkness word ,never show us how is him going

**Comment by rqhua** [comment]
I just quietly watching you

**Comment by Sirlu** [comment]
คุณถึงเตะไม่พระเจ้าทำไม ?

**Comment by FYH-fly** [comment]
it's so good, are you use the github's issues? I think you may be is the first use it.

**Comment by Sirlu** [comment]
Ich sehe, du hast die rinder in den Himmel

**Comment by FYH-fly** [comment]
/** 
* 
*----------Dragon be here!----------/ 
* 　　　┏┓　　　┏┓ 
* 　　┏┛┻━━━┛┻┓ 
* 　　┃　　　　　　　┃ 
* 　　┃　　　━　　　┃ 
* 　　┃　┳┛　┗┳　┃ 
* 　　┃　　　　　　　┃ 
* 　　┃　　　┻　　　┃ 
* 　　┃　　　　　　　┃ 
* 　　┗━┓　　　┏━┛ 
* 　　　　┃　　　┃神兽保佑 
* 　　　　┃　　　┃代码无BUG！ 
* 　　　　┃　　　┗━━━┓ 
* 　　　　┃　　　　　　　┣┓ 
* 　　　　┃　　　　　　　┏┛ 
* 　　　　┗┓┓┏━┳┓┏┛ 
* 　　　　　┃┫┫　┃┫┫ 
* 　　　　　┗┻┛　┗┻┛ 
* ━━━━━━神兽出没━━━━━━ 
*/

**Comment by rqhua** [comment]
The value of the issues for me that is I can talking with you ,what a meaningful issue!

**Comment by Sirlu** [comment]
[http://www.souid.com/](url)

**Comment by FYH-fly** [comment]
o yes, it's so good, I like you, I think every maybe do an important thing, that's to write blog, whether Technology or idears, it's can improve your's ability, and can change your feture way.

**Comment by rqhua** [comment]
It's not a web page you are looking for

**Comment by rqhua** [comment]
By writing ,you can make yourself clearly of what you are doing and working on , someday later,you will found you have made great improve.

**Comment by FYH-fly** [comment]
yes, I have begin try to write, but it's so terrible, I can't write well, the blog it's low bi

**Comment by rqhua** [comment]
@1187981465 haha , just try and you will be the best !

**Comment by FYH-fly** [comment]
@xiaocaifly 嘿嘿嘿

**Comment by rqhua** [comment]
Everyday,I work on many things ,but in my options,they are not important and i don't know whether it's meaningful or not.

**Comment by FYH-fly** [comment]
@xiaocaifly On the contrary, my work is even more boring. I don't know what I do everyday.  on the working time I only surf on the Internet，and write 200 ~ 300 lines C++ code，I feel so quick，just two hours later， Then you'll be free. I'll be free， and think so happy

**Comment by FYH-fly** [comment]
@xiaocaifly c++ code is so terrible，I can't put up with it.
